pkg/cmd/server/api/validation/master.go

```diff
@@ -654,7 +654,7 @@ func ValidateAPIServerExtendedArguments(config api.ExtendedArguments, fldPath *f
 }
 
 func ValidateControllerExtendedArguments(config api.ExtendedArguments, fldPath *field.Path) field.ErrorList {
-	return ValidateExtendedArguments(config, controlleroptions.NewCMServer().AddFlags, fldPath)
```

Comment:
We will see when we do the server wiring. Maybe we keep my approach, maybe we replace it. I would keep this here in the PR as it also changes all other place in the code where it's relevant.